### DISCUSSION TITLE
About 2D cropped images in examples.zip

### DISCUSSION THREAD

**Discussion by ZhangRenkai** [Nicholasli1995/EvoSkeleton#72] [Q&A] [answered]
Thanks for your great work. I noticed that the 2D HRNet estimate step estimates 2D coordinates using cropped images (in the examples.zip file) , and I wonder where the cropped anchors come from, are they produced using an pretrained 2D detector or just generated by 2D GT labels. If an 2D detector is used, could you please provide the message or pretrained models download link of the detector model. 
Another detail is whether the cropped images just reshaped from anchor size to HRNet input size(384*288) and there's no other process.
Thank you so much!!!

**Comment by Nicholasli1995** [comment, marked as answer]
> Thanks for your great work. I noticed that the 2D HRNet estimate step estimates 2D coordinates using cropped images (in the examples.zip file) , and I wonder where the cropped anchors come from, are they produced using an pretrained 2D detector or just generated by 2D GT labels. If an 2D detector is used, could you please provide the message or pretrained models download link of the detector model. Another detail is whether the cropped images just reshaped from anchor size to HRNet input size(384*288) and there's no other process. Thank you so much!!!

1. The GT bounding boxes are used for cropping. This is because the noise of using a detector does not affect the performance as suggested by previous studies using CPN.

2. The bounding box is simply reshaped into the used aspect ratio. No further complex pre-processing is used.
